discovery-infra/test_infra/utils/env_variables_utils.py

```diff
@@ -0,0 +1,87 @@
+    worker_disk_count: int = int(get_env(EnvVarsKeys.WORKER_DISK_COUNT, resources.DEFAULT_DISK_COUNT))
+    storage_pool_path: str = get_env(EnvVarsKeys.STORAGE_POOL_PATH, env_defaults.DEFAULT_STORAGE_POOL_PATH)
+    cluster_name: ClusterName = ClusterName()
+    private_ssh_key_path: str = get_env(EnvVarsKeys.PRIVATE_KEY_PATH, env_defaults.DEFAULT_SSH_PRIVATE_KEY_PATH)
+    kubeconfig_path: str = get_kubeconfig_path(cluster_name)
+    installer_kubeconfig_path: str = get_env(
+        EnvVarsKeys.INSTALLER_KUBECONFIG, env_defaults.DEFAULT_INSTALLER_KUBECONFIG
+    )
+    log_folder: str = get_env(EnvVarsKeys.LOG_FOLDER, env_defaults.DEFAULT_LOG_FOLDER)
+    service_network_cidr: str = get_env(EnvVarsKeys.SERVICE_CIDR, env_defaults.DEFAULT_SERVICE_CIDR)
+    cluster_network_cidr: str = get_env(EnvVarsKeys.CLUSTER_CIDR, env_defaults.DEFAULT_CLUSTER_CIDR)
+    cluster_network_host_prefix: int = int(get_env(EnvVarsKeys.HOST_PREFIX, env_defaults.DEFAULT_HOST_PREFIX))
+    is_static_ip: bool = bool(strtobool(
+        get_env(EnvVarsKeys.IS_STATIC_IP, default=str(env_defaults.DEFAULT_IS_STATIC_IP))))
+    iso_image_type: str = get_env(EnvVarsKeys.ISO_IMAGE_TYPE, env_defaults.DEFAULT_IMAGE_TYPE)
+    worker_vcpu: str = get_env(EnvVarsKeys.WORKER_CPU, resources.DEFAULT_WORKER_CPU)
+    master_vcpu: str = get_env(EnvVarsKeys.MASTER_CPU, resources.DEFAULT_MASTER_CPU)
+    test_teardown: bool = bool(strtobool(get_env(EnvVarsKeys.TEST_TEARDOWN, str(env_defaults.DEFAULT_TEST_TEARDOWN))))
+    namespace: str = get_env(EnvVarsKeys.NAMESPACE, consts.DEFAULT_NAMESPACE)
+    olm_operators: List[str] = field(default_factory=list)
+    platform: str = get_env(EnvVarsKeys.PLATFORM, env_defaults.DEFAULT_PLATFORM)
+    user_managed_networking: bool = env_defaults.DEFAULT_USER_MANAGED_NETWORKING
+    high_availability_mode: str = env_defaults.DEFAULT_HIGH_AVAILABILITY_MODE
+    download_image: bool = bool(
+        strtobool(get_env(EnvVarsKeys.DOWNLOAD_IMAGE, str(env_defaults.DEFAULT_DOWNLOAD_IMAGE))))
+    is_ipv6: bool = bool(strtobool(get_env(EnvVarsKeys.IS_IPV6, str(env_defaults.DEFAULT_IS_IPV6))))
+    cluster_id: str = get_env(EnvVarsKeys.CLUSTER_ID)
+    additional_ntp_source: str = get_env(EnvVarsKeys.ADDITIONAL_NTP_SOURCE, env_defaults.DEFAULT_ADDITIONAL_NTP_SOURCE)
+    network_name: str = get_env(EnvVarsKeys.NETWORK_NAME, env_defaults.DEFAULT_NETWORK_NAME)
+    bootstrap_in_place: bool = bool(strtobool(
+        get_env(EnvVarsKeys.BOOTSTRAP_IN_PLACE, str(env_defaults.DEFAULT_BOOTSTRAP_IN_PLACE))))
+    single_node_ip: str = get_env(EnvVarsKeys.SINGLE_NODE_IP, env_defaults.DEFAULT_SINGLE_NODE_IP)
+    worker_cpu_mode: str = get_env(EnvVarsKeys.WORKER_CPU_MODE, env_defaults.DEFAULT_TF_CPU_MODE)
+    master_cpu_mode: str = get_env(EnvVarsKeys.MASTER_CPU_MODE, env_defaults.DEFAULT_TF_CPU_MODE)
+    iso_download_path: str = get_env(EnvVarsKeys.ISO_DOWNLOAD_PATH)
```

Comment:
Could we have a sweet generation function for this? I really don't like having 3 modules having the same giant list